devtools/platform-descriptor-json/src/main/resources/templates/basic-rest/java/build.gradle-template.ftl

```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 repositories {
-     mavenLocal()
+     mavenLocal()${maven_repositories}
```

Comment:
ordering matters - mavencentral should come before red hat maven to not get super slow for artifacts that are from mavencentral.

Reply:
@maxandersen already merged ^^ I'll push a fix